### ISSUE TITLE
Change Model: Pure Conve

### ISSUE THREAD

**Issue by hackerekcah** [hackerekcah/asc-key-events#1]
[construct_cnnL3_7_smp](https://github.com/marl/milsed/blob/master/milsed/models.py)

**Comment by hackerekcah** [comment]
# -- Pure convolution models
def construct_cnnL3_7_smp(pump, alpha):
    '''
    Like crnnL3_5 but without gru layers
    Parameters
    ----------
    pump
    alpha
    Returns
    -------
    '''
    model_inputs = ['mel/mag']

    # Build the input layer
    layers = pump.layers()

    x_mag = layers['mel/mag']

    # Apply batch normalization
    x_bn = K.layers.BatchNormalization()(x_mag)

    # x_sq = milsed.layers.SqueezeLayer()(x_bn)

    # BLOCK 1
    conv1 = K.layers.Convolution2D(16, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(x_bn)
    bn1 = K.layers.BatchNormalization()(conv1)
    conv2 = K.layers.Convolution2D(16, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(bn1)
    bn2 = K.layers.BatchNormalization()(conv2)
    pool2 = K.layers.MaxPooling2D((2,2), padding='valid')(bn2)

    # BLOCK 2
    conv3 = K.layers.Convolution2D(32, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(pool2)
    bn3 = K.layers.BatchNormalization()(conv3)
    conv4 = K.layers.Convolution2D(32, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(bn3)
    bn4 = K.layers.BatchNormalization()(conv4)
    pool4 = K.layers.MaxPooling2D((2, 2), padding='valid')(bn4)

    # BLOCK 3
    conv5 = K.layers.Convolution2D(64, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(pool4)
    bn5 = K.layers.BatchNormalization()(conv5)
    conv6 = K.layers.Convolution2D(64, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(bn5)
    bn6 = K.layers.BatchNormalization()(conv6)
    pool6 = K.layers.MaxPooling2D((2, 2), padding='valid')(bn6)

    # BLOCK 4
    conv7 = K.layers.Convolution2D(128, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(pool6)
    bn7 = K.layers.BatchNormalization()(conv7)
    conv8 = K.layers.Convolution2D(128, (3, 3),
                                   padding='same',
                                   activation='relu',
                                   kernel_initializer='he_normal')(bn7)
    bn8 = K.layers.BatchNormalization()(conv8)
    pool8 = K.layers.MaxPooling2D((2, 2), padding='valid')(bn8)

    # CONV SQUEEZE
    conv_sq = K.layers.Convolution2D(256, (1, 8),
                                     padding='valid',
                                     activation='relu',
                                     kernel_initializer='he_normal')(pool8)
    bn8 = K.layers.BatchNormalization()(conv_sq)
    sq2 = milsed.layers.SqueezeLayer(axis=-2)(bn8)

    n_classes = pump.fields['static/tags'].shape[0]

    p0 = K.layers.Dense(n_classes, activation='sigmoid',
                        bias_regularizer=K.regularizers.l2())

    p_dynamic = K.layers.TimeDistributed(p0, name='dynamic/tags')(sq2)

    p_static = milsed.layers.SoftMaxPool(alpha=alpha,
                                         axis=1,
                                         name='static/tags')(p_dynamic)

    model = K.models.Model([x_mag],
                           [p_dynamic, p_static])

    model_outputs = ['dynamic/tags', 'static/tags']

    return model, model_inputs, model_outputs
